flagscale/train/train_qwen2_5_vl.py

```diff
@@ -758,17 +799,49 @@ def add_multimodal_extra_args(parser):
     group.add_argument("--temporal-patch-size", type=int, default=2)
     group.add_argument("--patch-size", type=int, default=14)
     group.add_argument("--max-padding-length", type=int, default=2048)
-    group.add_argument("--enable-variable-seq-lengths", action="store_true", default=False, help="Enable variable sequence lengths")
-    group.add_argument("--vision-root", type=str, default = None, help="The vision dirctory root path.")
-    group.add_argument("--max-samples-per-sequence", type=int, default=2**31-1, help="max sequencial seqence samples in a slice")
-    group.add_argument("--shuffle-buffer-size", type=int, default=0, help="the buffer size to shuffle the samples in a seqence")
+    group.add_argument(
+        "--enable-variable-seq-lengths",
+        action="store_true",
+        default=False,
+        help="Enable variable sequence lengths",
+    )
+    group.add_argument(
+        "--vision-root", type=str, default=None, help="The vision dirctory root path."
+    )
+    group.add_argument(
+        "--max-samples-per-sequence",
+        type=int,
+        default=2**31 - 1,
+        help="max sequencial seqence samples in a slice",
```

Comment:
please be consistent when deciding whether capitalize the first character in the help message.